src/api/app/controllers/webui/packages/bs_requests_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Webui
+  module Packages
+    class BsRequestsController < WebuiController
+      def index
+        package = Package.find_by(name: params[:package])
```

Comment:
It seems `require_package` requires the :project param to be set as well? But current I have the route set up as this:
`/packages/:package/requests.json`

Should I change the route so that it takes both a project and package name?